src/Microsoft.DotNet.ProjectModel/Files/PatternGroup.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.DotNet.ProjectModel.Files
 {
     public class PatternGroup
     {
+        private static readonly Dictionary<string, IEnumerable<string>> s_resolvedFilesCache = new Dictionary<string, IEnumerable<string>>();
```

Comment:
As long as it doesn't regress the performance for project model server.
/cc @abpiskunov